spec/pushmi_pullyu/preservation_queue_spec.rb

```diff
@@ -8,20 +8,24 @@
 
     before do
       PushmiPullyu.server_running = true
-      redis.zadd 'test:pmpy_queue', 1, 'noid1'
-      redis.zadd 'test:pmpy_queue', 3, 'noid3'
-      redis.zadd 'test:pmpy_queue', 4, 'noid2'
```

Comment:
Are `noid1`, `noid2`, and `noid3` still used or defined somewhere?

Reply:
These values where just used for this test.